runtime/core/core/test/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+set(TARGET_NAME ov_core_unit_tests)
```

Comment:
@ilyachur what do you think about naming in camelCase style? At the moment the most OV tests named like that (cpuFuncTests etc.). Also ov_ prefix looks redundant

Reply:
I used this style by two reasons:

- We are using snake case for our libraries and in our code style.
- core is a common name, it is why we need to have ov prefix.